plugins/emojifyme/emojifyme.go

```diff
@@ -71,19 +71,25 @@ func (p *EmojifyMePlugin) Message(message msg.Message) bool {
 	tokens := strings.Fields(strings.ToLower(message.Body))
 	for i, token := range tokens {
 		if _, ok := p.Emoji[token]; ok {
-			emojied++
+			if token != "a" && token != "it" {
```

Comment:
Read these strings in from the global config stuff, and then stick them into a map and check them that way.

Reply:
I thought you'd like hardcoding more… 😈